setup.py

```diff
@@ -10,7 +10,7 @@
     description="Hierarchical classification interface extensions for scikit-learn",
     author="Globality Engineering",
     author_email="engineering@globality.com",
-    url="https://github.com/globality-corp/sklearn-hierarchical-classification",
+    url="https://github.com/globality-corp/sklearn-hierarchical",
```

Comment:
Did you change the name of the project? Did you change it in PyPI? Etc.

Reply:
Seems like we should change the repo name if we change the rest of the package, no?